docs/usage/2-manifest.md

```diff
@@ -88,6 +107,39 @@ Each target in the list of project targets can be initialized with the following
   - **CurrentVersion:** Current version without the extension.
 - **Environment (optional):** It's a `[String: String]` to defined variables that will be set to the scheme that Tuist automatically generates for the target.
 
+### Scheme
+
+A `Scheme` will help to define a collection of targets to `Build, Run, Test, Profile, Analyze and Archive`:
+> Note: Tuist will auto-generate Scheme for each target if they are not defined. 
+- **name: String:** Name of Scheme
+- **shared: Bool:** Scheme is shared or not 
+- **buildAction: BuildAction?:** Scheme Build options
+    - **targets: [String]:** A list of targets will be built, which are defined in Project
+    - **preActions:[ [ExecutionAction](#execution-action)]:** A list of pre build actions
+    - **postActions: [[ExecutionAction](#execution-action)]:** A list of post build actions
+- **testAction: TestAction?:** Scheme Test options
+    - **targets: [String]:** A list of targets will be tested, which are defined in Project
+    - **arguments: [Arguments?](#arguments):** Commandline arguments passed on launch and Environment variables
+    - **config: BuildConfiguration:** Test with  `debug` or `release` configuration
+    - **coverage: Bool:** Scheme should gather coverage data or not. Default is `false`
+    - **preActions: [[ExecutionAction](#execution-action)]:** Pre test script action
+    - **postActions: [[ExecutionAction](#execution-action)]:** Post test script action
+- **runAction: RunAction?:** Run or Launch action
+    - **config: BuildConfiguration:** Run with `debug` or `release` configuration
+    - **executable: String?:** Executable or target will be launch
+    - **arguments: [Arguments?](#arguments):** Commandline arguments passed on launch and Environment variables
+    
+#### Execution Action
+Scheme run scripts can be defined with the following attributes:
+- **title: String:** Name of a script. Defaults to `Run Script`
+- **scriptText: String:** An inline shell script
+- **target: String?:** Name of a build or test target that will provide build settings.
```

Comment:
In which scenarios can this attribute be optional? I'd give it a default value, `nil`.